Pending_updates/com_017_major_ports/README.md

```diff
@@ -0,0 +1,12 @@
+## Major Ports Dataset Pre-processing
+This file describes the data pre-processing that was done to [the World Port Index](https://stats.oecd.org/Index.aspx?DataSetCode=MUNW) for [display on Resource Watch](https://resourcewatch.org/data/explore/31976299-3bd7-4926-acb0-8daadf7d9863).
+
+The data source provided the dataset as one shapefile and one csv file. This data file was not modified from the original version for display on Resource Watch.
```

Comment:
You will want to specify which of these files you used for the processing. I see that the csv was used, but we will need the_geom column in order to display it on RW. I would recommend changing this to use the shapefile, or generating the column for the_geom by using the lat and long and turning them into a geojson.